Assets/MRTK/Examples/StandardAssets/Prefabs/Menus.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 2d722e77d29e60349b2d1f9cd9edc68c
+guid: 7088112ab8268e14aa4218c57abd997d
```

Comment:
Why did this GUID change?

Reply:
Hmm, not sure. Maybe project was opened in later 2018 version and just commited. I'll look at reverting